src/2_add_cots_priority.jl

```diff
@@ -0,0 +1,51 @@
+using
+    CSV,
+    Dates,
+    DataFrames
+
+using
+    GLMakie,
+    GeoMakie
+
+using
+    Statistics,
+    Bootstrap
+
+import GeoDataFrames as GDF
+import GeoFormatTypes as GFT
+import ArchGDAL as AG
+
+include("common.jl")
+
+# Load required data.
+RRAP_lookup = GDF.read(joinpath(OUTPUT_DIR,find_latest_file(OUTPUT_DIR)))
```

Comment:
Standardize spacing please

```suggestion
RRAP_lookup = GDF.read(joinpath(OUTPUT_DIR, find_latest_file(OUTPUT_DIR)))
```